react/components/EditorContainer/Sidebar/ConfigurationList/index.tsx

```diff
@@ -70,13 +71,40 @@ defineMessages({
   },
 })
 
+const omitUndefined = pickBy(val => typeof val !== 'undefined')
+
 class ConfigurationList extends React.Component<Props, State> {
   private component: Extension['component']
   private componentImplementation: RenderComponent<any, any> | null
   private componentProperties: ComponentSchema['properties']
   private componentTitle: ComponentSchema['title']
   private contentSchema: JSONSchema6
 
+  private handleFormChange: FormProps<object>['onChange'] = debounce(event => {
```

Comment:
:pick: Why is a private method being defined before the constructor?

Reply:
Because it is a variable :|